packages/core/resolve-runtime/src/common/handlers/deploy-service-event-handler.js

```diff
@@ -1,30 +1,56 @@
 import bootstrap from '../bootstrap'
+import { RESOLVE_SAGA_PREFIX } from '../sagas/constants'
+
+const getReadModelNames = resolve => {
+  return resolve.readModels
+    .map(({ name }) => name)
+    .filter(name => !(name.indexOf(RESOLVE_SAGA_PREFIX) === 0))
```

Comment:
Why not `.filter(name => name.indexOf(RESOLVE_SAGA_PREFIX) !== 0)`?
This check will contain scheduler saga.